Source/WebCore/Modules/applepay/ApplePaySessionPaymentRequest.h

```diff
@@ -167,7 +167,90 @@ class ApplePaySessionPaymentRequest {
     const std::optional<ApplePayLaterAvailability>& applePayLaterAvailability() const { return m_applePayLaterAvailability; }
     void setApplePayLaterAvailability(const std::optional<ApplePayLaterAvailability>& applePayLaterAvailability) { m_applePayLaterAvailability = applePayLaterAvailability; }
 #endif
-    
+
+    ApplePaySessionPaymentRequest(String&& countryCode,
+        String&& currencyCode,
+        struct ContactFields&& requiredBillingContactFields,
```

Comment:
I don't think struct is necessary here.  Same with MerchantCapabilities

Reply:
Fixed